css/styles.css

```diff
@@ -0,0 +1,66 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+ul {
+  scrollbar-color: gray rgb(252, 252, 252);
+  scrollbar-width: thin;
+}
+
+li {
+  cursor: pointer;
+  padding: 7.3px;
+  list-style-type: none;
+  margin-top: 0;
+  display: flex;
+}
+
+.marker {
+  width: 22px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.pokedex-list {
+  overflow: scroll;
+  font-size: 12.5px;
```

Comment:
Per [effective-css](https://lms.learningfuze.com/code-guides/Learning-Fuze/curriculum/effective-css), use rem for font-size!